cmake/kvi_sysbuildinfo.h.cmake

```diff
@@ -17,5 +17,7 @@
 #define KVIRC_BUILD_COMPILER "${CMAKE_KVIRC_BUILD_COMPILER}"
 #define KVIRC_BUILD_COMPILER_FLAGS "${CMAKE_KVIRC_BUILD_COMPILER_FLAGS}"
 #define KVIRC_BUILD_REVISION "${CMAKE_KVIRC_BUILD_REVISION}"
+#define KVIRC_SOURCES_DATE_GIT "${CMAKE_KVIRC_SOURCES_DATE}"
+#define KVIRC_SOURCES_DATE_GIT_HEX "${CMAKE_KVIRC_SOURCES_DATE_HEX}"
```

Comment:
To fix the issue, just remove the quotes from this line, so that the #define will get an actual hex value and not a string.